packages/framer-motion/src/value/types/utils.ts

```diff
@@ -9,9 +9,9 @@ export const sanitize = (v: number) => Math.round(v * 100000) / 100000
 
 export const floatRegex = /(-)?([\d]*\.?[\d])+/g
 export const colorRegex =
-    /(#[0-9a-f]{6}|#[0-9a-f]{3}|#(?:[0-9a-f]{2}){2,4}|(rgb|hsl)a?\((-?[\d\.]+%?[,\s]+){2}(-?[\d\.]+%?)\s*[\,\/]?\s*[\d\.]*%?\))/gi
+    /(#[0-9a-f]{2,8}|(rgb|hsl)a?\((-?[\d\.]+%?[,\s]+){2}(-?[\d\.]+%?)\s*[\,\/]?\s*[\d\.]*%?\))/gi
```

Comment:
Why is this `8` and not `7`?
I would assume that it can be 3 (`#fff`), `4` (`#fff0`) `6` (`#00aaff`) and `7` (`#00aaff0`)

Reply:
So when exactly is it `2` or `8`?

Reply:
Ah, the opacity can be two hex digits too.

Reply:
But then the minimum would still be `3`, right?

Reply:
Ah yeah it is 3

Reply:
(8 being 2 digit alpha)